.ci/macos/dependencies/gource.sh

```
@@ -0,0 +1,118 @@
+#  ./install-sh -c -d '/usr/local/lib/pkgconfig'
+#  /usr/bin/install -c -m 644 SDL2_image.pc '/usr/local/lib/pkgconfig'
+# ----------------------------------------------------------------------
+# ========================================================================================================
+
+# install dependencies boost
+# ========================================================================================================
+# Install Boost
+brew install boost
+
+# Set BOOST_ROOT environment variable
+export BOOST_ROOT=$(brew --prefix boost)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)